optaplanner-benchmarks/pom.xml

```diff
@@ -1,20 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <parent>
-    <artifactId>kie-benchmarks</artifactId>
-    <groupId>org.kie</groupId>
-    <version>7.57.0-SNAPSHOT</version>
-  </parent>
-  <modelVersion>4.0.0</modelVersion>
 
   <groupId>org.kie</groupId>
   <artifactId>optaplanner-benchmarks</artifactId>
   <packaging>pom</packaging>
+  <version>1.0-SNAPSHOT</version>
+
+  <modelVersion>4.0.0</modelVersion>
+
   <name>OptaPlanner Benchmarks Parent</name>
   <description>This is the multi-module project top-level parent.</description>
 
+  <properties>
+    <version.org.optaplanner>8.1.0.Final</version.org.optaplanner>
```

Comment:
Why this particular version? The last one is 8.7.0.Final. Also, keep in mind that the weekly benchmarks from the master branch have been so far relying on having the right version coming from the kie-parent. Unless you make this version get updated with every release, it will get stale again.

Reply:
I am thinking if this is the right place, if rest of the kie-benchmarks are meant to stay on 7.x. Is there any repository for Kogito performance tests?

Reply:
1. I set this version for a default local launching. The idea is to change it in the pipeline and pass it here. So far the fastest way would be to relay on the build event from eng jenkins an get the new version of OptaPlanner from there. 

2. The problem is that Kogito deligated the whole regression automation to the eng. Look here:https://docs.google.com/document/d/1jW504qzE78ernheBKjHo00MP-TZPanqR9iPaH3rAeo8/edit#heading=h.g4tx0m2x25rd

Reply:
I think that maybe it is fine for their specific problem, but for calculation OptaPlanner needs regression checks.

Reply:
@dupliaka unfortunately I don't have access to that document. Anyway, if every pipeline that is supposed to run the benchmarks will override the version, it's ok.